src/data/markdown/docs/02 javascript api/09 k6-net-grpc.md

```diff
@@ -0,0 +1,43 @@
+
+<Blockquote mod="warning">
+
+The k6 gRPC API is currently considered in beta and is subject to change. Future k6 versions might have slight differences in the method and type signatures described in this documentation.
+
+</Blockquote>
+
+| Class/Method | Description |
+|--------------|-------------|
+| [Client](/javascript-api/k6-net-grpc/client) | gRPC client used for making RPC calls to a gRPC Server |
+| [Client.load(importPaths, ...protoFiles)](/javascript-api/k6-net-grpc/client/client-load-importpaths----protofiles) | Loads and parses the given protocol buffer definitions to be made available for RPC requests. |
+| [Client.connect(address [,params])](/javascript-api/k6-net-grpc/client/client-connect-address-params) | Connects to a given gRPC service. |
+| [Client.invoke(url, request [,params])](/javascript-api/k6-net-grpc/client/client-invoke-url-request-params) | Makes an unary RPC for the given service/method and returns a [Response](/javascript-api/k6-net-grpc/response). |
+| [Client.close()]() | Close the connection to the gRPC service. |
+| [Params](/javascript-api/k6-net-grpc/params) | RPC Request specific options. |
+| [Response](/javascript-api/k6-net-grpc/response) | Returned by RPC requests. |
+
+| Constant | Description |
+|----------|-------------|
+| `StatusOK` | OK is returned on success. |
+| `StatusCanceled` | Canceled indicates the operation was canceled (typically by the caller). |
+| `StatusUnknown` | Unknown error. |
+| `StatusInvalidArgument` | InvalidArgument indicates client specified an invalid argument. |
+| `StatusDeadlineExceeded` | DeadlineExceeded means operation expired before completion. |
+| `StatusNotFound` | NotFound means some requested entity (e.g., file or directory) was not found. |
+| `StatusAlreadyExists` | AlreadyExists means an attempt to create an entity failed because one already exists. |
+| `StatusPermissionDenied` | PermissionDenied indicates the caller does not have permission to execute the specified operation. |
+| `StatusResourceExhausted` | ResourceExhausted indicates some resource has been exhausted, perhaps a per-user quota, or perhaps the entire file system is out of space. |
+| `StatusFailedPrecondition` | FailedPrecondition indicates operation was rejected because the system is not in a state required for the operation's execution. |
+| `StatusAborted` | Aborted indicates the operation was aborted, typically due to a concurrency issue like sequencer check failures, transaction aborts, etc. |
+| `StatusOutOfRange` | OutOfRange means operation was attempted past the valid range. E.g., seeking or reading past end of file. |
+| `StatusUnimplemented` | Unimplemented indicates operation is not implemented or not supported/enabled in this service. |
+| `StatusInternal` | Internal errors. Means some invariants expected by underlying system has been broken. If you see one of these errors, something is very broken. |
```

Comment:
Don't think the second sentence adds any value, though we should probably mention a few examples of the kinds of issues that cause this.

```suggestion
| `StatusInternal` | Internal errors. Means some invariants expected by the underlying system have been broken. |
```